### PR TITLE
Improve lists kartlag infobox

### PR DESCRIPTION
**Pull request by kepatinin**
- Changed the close/open button on the layer menu (kartlag)
- Adjusting icon size in layer menu so they look better
- Icons on infobox are the same as the layer
- Center the circular progress in infobox
- Add badge with exclamation mark in infobox when GetFeatureInfo API returns error

### CODE REVIEW

File: src/Common/CustomIcon.js [hunk 1]
```diff
@@ -12,7 +20,15 @@ const CustomIcon = ({ icon, size, styling, color, id, tooltipText }) => {
           placement="left"
           title={tooltipText}
         >
-          <div id={id || "id"} className="layer-icon-wrapper">
+          <div
+            id={id || "id"}
```
Comment (bjornreppen) [inline src/Common/CustomIcon.js:24]: Duplicated block

File: src/Forvaltningsportalen/FeatureInfo/GeneriskElement.js [hunk 2]
```diff
@@ -37,20 +38,29 @@ const GeneriskElement = props => {
...
-            <CircularProgress />
+            <CircularProgress size={30} />
           ) : (
             <>
               {resultat.error ? (
-                <ErrorOutline />
+                <Badge badgeContent="!" color="primary" overlap="circle">
+                  <CustomIcon
+                    id="infobox-list-icon"
+                    icon={kartlag.tema}
```
Comment (bjornreppen) [inline src/Forvaltningsportalen/FeatureInfo/GeneriskElement.js:50]: Duplicated block

File: src/Forvaltningsportalen/ForvaltningsKartlag/ForvaltningsElement.js [hunk 3]
```diff
@@ -98,15 +105,17 @@ const ForvaltningsElement = ({
                   <CustomIcon
                     id="kartlag"
                     icon={kartlag.tema}
-                    size={30}
+                    size={isLargeIcon(kartlag.tema) ? 30 : 26}
```
Comment (bjornreppen) [inline src/Forvaltningsportalen/ForvaltningsKartlag/ForvaltningsElement.js:108]: Duplicated block

File: src/Forvaltningsportalen/KartlagFanen.js [hunk 4]
```diff
@@ -15,14 +17,20 @@ const KartlagFanen = props => {
...
+          variant="contained"
+          size="small"
           onClick={() => {
             props.toggleSideBar();
           }}
         >
-          {props.showSideBar ? "Skjul kartlag" : "Vis kartlag"}
-        </button>
+          {props.showSideBar ? (
+            <CustomIcon id="show-layers-icon" icon="menu-right" size={30} />
```
Comment (bjornreppen) [inline src/Forvaltningsportalen/KartlagFanen.js:29]: Duplicated block

**Comment by kepatinin** [comment]
I have now made changes based on your comments:
- Remove duplicated code.
- Made "layers" icon larges and moved so it fits with the other icons.
- List element is not changed to white background if there is no "faktaark".

**Pull request merged**